servant-docs/example/greet.hs

```diff
@@ -53,6 +53,14 @@ instance ToParam (MatrixParam "lang" String) where
                   "Get the greeting message selected language. Default is en."
                   Normal
 
+instance ToSample () Greet where
```

Comment:
I don't think this is doing the right thing. `ToSample` should pretty much always have it's first and second arguments the same (there is only one exception, `Headers ls a`, which is kind of sticking out like a sore thumb right now and caused us to add the second param in the first place). In essence, the first argument is what this is a sample for, and the second is what the type of the sample is (this might need to be better documented -- sorry). So in this case, you probably want `ToSample () ()`, or *maybe* another special case, `ToSample () String` with toSample ~~ "NO CONTENT".